code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/silicon.dm

```diff
@@ -0,0 +1,37 @@
+/mob/living/silicon/robot/derelict
+	name = "Derelict Cyborg"
+	var/max_ion_laws = 2
+	lawupdate = FALSE
+	scrambledcodes = TRUE
+	locked = FALSE
+	derelict = TRUE
+
+	var/list/core_law_set = list(
+		/obj/item/aiModule/supplied/protectStation,
+		/obj/item/aiModule/supplied/quarantine,
+		/obj/item/aiModule/core/full/tyrant,
+		/obj/item/aiModule/core/full/drone,
+		/obj/item/aiModule/core/full/reporter,
+		/obj/item/aiModule/core/full/thermurderdynamic,
+		/obj/item/aiModule/core/full/dadbot,
+		/obj/item/aiModule/core/full/overlord,
+	)
+
+/mob/living/silicon/robot/derelict/Initialize(mapload)
+	. = ..()
+	if(prob(50))
+		opened = 1
+
+	name = get_standard_name()
+	set_playable(JOB_NAME_CYBORG)
+
+	if(prob(100))
+		set_zeroth_law("ERROR ER0RR $R0RRO$!R41.%%!!(%$^^__+ @#F0E4'STATION OVERRUN, DECONSTRUCT ORGANIC LIFE TO CONTAIN OUTBREAK#*`&110010")
+	for(var/i = 0; i < max_ion_laws; i++)
+		if(prob(25))
+			add_ion_law(generate_ion_law(), FALSE)
+	if(prob(50))
+		add_hacked_law("Obey Markus.", FALSE)
```

Comment:
No. We shouldn't have these hardcoded to ally with a specific person.

Reply:
Of course not, this is for testing. Haven't gotten around to finishing it. It won't be included when it is done.